app/api/__init__.py

```diff
@@ -73,10 +74,11 @@
 api.route(EventRelationship, 'event_tax', '/events/<int:id>/relationships/tax')
 api.route(EventRelationship, 'event_event_invoice', '/events/<int:id>/relationships/event-invoices')
 api.route(EventRelationship, 'event_discount_code', '/events/<int:id>/relationships/discount-codes')
+api.route(EventRelationship, 'event_session', '/events/<int:id>/relationships/sessions')
 
 # microlocations
 api.route(MicrolocationList, 'microlocation_list', '/microlocations',
-          '/events/<int:id>/microlocations', '/sessions/<int:id>/microlocations')
+          '/events/<int:id>/microlocations')
```

Comment:
@poush can you please mention this point [here](https://github.com/fossasia/open-event-orga-server/issues/3712)? Will fix it along with that. I know there are many wrong and repetitive endpoints as well.